LeafletExample/README.md

```diff
@@ -23,13 +23,15 @@ and Android
 cordova platform add android
 ```
 
-Add IndoorAtlas Cordova plugin
-```
-cordova plugin add https://github.com/IndoorAtlas/cordova-plugin.git
```

Comment:
Why was this removed?

Reply:
I tried following the instructions on a new project and it appears this is not really needed since the plugin is already defined in the `config.xml`